orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <queue>
+#include <string>
+
+#include "core/optimizer/graph_transformer.h"
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/compute_optimizer/shared_utils.h"
+
+namespace onnxruntime {
+
+/**
+@Class PaddingElimination
+
```

Comment:
Better to detailed explanation for what this graph transformer did, and how it tranform the graphs.